services/storage/src/simcore_service_storage/modules/celery/models.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any, TypeAlias
+from uuid import UUID
+
+from models_library.progress_bar import ProgressReport
+from pydantic import BaseModel
+
+TaskContext: TypeAlias = dict[str, Any]
+TaskID: TypeAlias = str
+TaskUUID: TypeAlias = UUID
+
+
+class TaskStatus(BaseModel):
```

Comment:
is this a task progress status? I am not sure I understand the use-case of having the progress optional